### ISSUE TITLE
[Feature Request] Multi Users playlists

### ISSUE THREAD

**Issue by BuckTheWeasel** [Kaiede/PlexPlaylistHost#5]
hello there !

Thanks for help at issue #4  Also got it working now.

however i have on my account also 4 users 
but i can't see playlists from other users only on my admin account..
so is there maybe a fix for that or a workaround ? 

Suggestion:
maybe a selection with users to access at music channels playlist ?

anyway thanks for your help,

Buck.

**Comment by Kaiede** [comment]
This is with managed or home users?

If so, it explains why playlists live at the server level, instead of library. Hmm.

Unfortunately, these sort of users are not something I'm too familiar with in the API.

**Comment by Kaiede** [comment]
This said, I will note that this project is something I stopped using myself a while ago. Too many oddities in the UPnP layer of Plex turned me off from using it full time. But I'm happy to take pull requests/etc.

If I had to revisit this, I'd probably do it as external UPnP AV server that uses Plex as the back-end.

**Comment by BuckTheWeasel** [comment]
> This is with managed or home users?

yes home users sadly and plex is ditching plugins so also that gonne be a problem :/

> This said, I will note that this project is something I stopped using myself a while ago. Too many oddities in the UPnP layer of Plex turned me off from using it full time. But I'm happy to take pull requests/etc.

will do ! if i find a fix for it i will make a pull request, Yea it really sad that plex not really doing anything with UPnP still today a lot of Media players still using it,
> 
> If I had to revisit this, I'd probably do it as external UPnP AV server that uses Plex as the back-end.

 that will be a great idea for music, Plex has plexamp but it cannot be used with Dac music player only with UPnP server and so many people have ask about it, 

But thanks for the help i will keep a eye on your external UPnP AV server (If you going to make it.)
Buck.
